source/new.rst

```diff
@@ -91,7 +91,7 @@ without having to rely on services from third parties.
 Our verification approach thus fits into the Autocrypt key distribution model
 which does not require extra services from third parties either.
 
-While Autocrypt Level 1 focusses on passive attacks
+While Autocrypt Level 1 focuses on passive attacks
```

Comment:
"focusses" seems to be okayish as well, however, in both US and UK, "focuses" seems to be always more in use since 50+ years